app/assets/stylesheets/components/concept_details.scss

```diff
@@ -6,4 +6,12 @@
 
 .concept_details_component table th {
   width: 220px;
+}
+
+.concept_details_component td, .concept_details_component th{
+  max-width: 200px;
+}
+#concepts_json_button{
```

Comment:
don't use an ID too specific. 
and this file is only for the styling of the concept details component. So does this apply

Reply:
This css is related only to the concepts json button inside the concept details component.

Reply:
so put it in `concepts.scss` file, no?